File: eggsy/src/main/java/com/spring/datasource/Lesson.java
package com.spring.datasource;

import java.util.Date;

import javax.xml.bind.annotation.XmlElement;
import javax.xml.bind.annotation.XmlRootElement;

@XmlRootElement(name = "lesson")
public class Lesson {
	private long id;
	private String name;
	private String description;
	private long isDeleted;
	private String createdBy;
	private String lastUpdatedBy;
	private Date createDate;
	private Date lastUpdateDate;
	private String eventDate;
	private String eventTime;
	private long lessonTypeId;
	private long userId;
	private String username;
	private String sessionId;
	private long isLive;
	private int rating;

	public long getId() {
		return id;
	}

	@XmlElement
	public void setId(long id) {
		this.id = id;
	}

	public String getName() {
		return name;
	}

	@XmlElement
	public void setName(String name) {
		this.name = name;
	}

	public String getDescription() {
		return description;
	}

	@XmlElement
	public void setDescription(String description) {
		this.description = description;
	}

	public long getIsDeleted() {
		return isDeleted;
	}

	@XmlElement
	public void setIsDeleted(long isDeleted) {
		this.isDeleted = isDeleted;
	}

	public String getCreatedBy() {
		return createdBy;
	}

	@XmlElement
	public void setCreatedBy(String createdBy) {
		this.createdBy = createdBy;
	}

	public String getLastUpdatedBy() {
		return lastUpdatedBy;
	}

	@XmlElement
	public void setLastUpdatedBy(String lastUpdatedBy) {
		this.lastUpdatedBy = lastUpdatedBy;
	}

	public Date getCreateDate() {
		return createDate;
	}

	@XmlElement
	public void setCreateDate(Date createDate) {
		this.createDate = createDate;
	}

	public Date getLastUpdateDate() {
		return lastUpdateDate;
	}

	@XmlElement
	public void setLastUpdateDate(Date lastUpdateDate) {
		this.lastUpdateDate = lastUpdateDate;
	}

	public long getLessonTypeId() {
		return lessonTypeId;
	}

	@XmlElement
	public void setLessonTypeId(long lessonTypeId) {
		this.lessonTypeId = lessonTypeId;
	}

	public long getUserId() {
		return userId;
	}

	@XmlElement
	public void setUserId(long userId) {
		this.userId = userId;
	}

	public String getSessionId() {
		return sessionId;
	}

	public void setSessionId(String sessionId) {
		this.sessionId = sessionId;
	}

	public String getUsername() {
		return username;
	}

	public void setUsername(String username) {
		this.username = username;
	}

	public String getEventDate() {
		return eventDate;
	}

	public void setEventDate(String eventDate) {
		this.eventDate = eventDate;
	}

	public String getEventTime() {
		return eventTime;
	}

	public void setEventTime(String eventTime) {
		this.eventTime = eventTime;
	}
	
	public long getIsLive() {
		return isLive;
	}

	public void setIsLive(long isLive) {
		this.isLive = isLive;
	}
	public long getRating() {
		return rating;
	}

	public void setRating(int rating) {
		this.rating = rating;
	}

}

File: hobbyhorse/Platform/Data/Lessons/Lesson.php
<?php
/**
 * Created by JetBrains PhpStorm.
 * User: radhika
 * Date: 4/3/12
 * Time: 12:46 AM
 * To change this template use File | Settings | File Templates.
 */
require_once("Platform/Data/Base.php");

class Platform_Data_Lessons_Lesson extends Platform_Data_Base
{
    /**
     * @var
     */
    protected $_lessonTypeId;
    /**
     * @var
     */
    protected $_userId;
/**
     * @var
     */
    protected $_username;/**
     * @var
     */
    protected $_eventDate;/**
     * @var
     */
    protected $_eventTime;
    
    /* @var
     */
    protected $_sessionId;

    /**
     * @return
     */

    public function __construct()
    {

    }


    public function getLessonTypeId()
    {
        return $this->_lessonTypeId;
    }

    /**
     * @param  $lessonTypeId
     */
    public function setLessonTypeId($lessonTypeId)
    {
        $this->_lessonTypeId = $lessonTypeId;
    }
    
    public function getUserId()
    {
        return $this->_userId;
    }

    /**
     * @param  $lessonTypeId
     */
    public function setUserId($userId)
    {
        $this->_userId = $userId;
    }
    
    public function getUsername()
    {
        return $this->_username;
    }

    /**
     * @param  $lessonTypeId
     */
    public function setUsername($username)
    {
        $this->_username = $username;
    }
    
    public function getEventDate()
    {
        return $this->_eventDate;
    }

    /**
     * @param  $eventDate
     */
    public function setEventDate($eventDate)
    {
        $this->_eventDate = $eventDate;
    }
    public function getEventTime()
    {
        return $this->_eventTime;
    }

    /**
     * @param  $lessonTypeId
     */
    public function setEventTime($eventTime)
    {
        $this->_eventTime = $eventTime;
    }
    
     public function getSessionId()
    {
        return $this->_sessionId;
    }

    /**
     * @param  $lessonTypeId
     */
    public function setSessionId($sessionId)
    {
        $this->_sessionId = $sessionId;
    }
    
}
